static/app/components/events/eventReplay/replayPreview.tsx

```diff
@@ -45,9 +48,24 @@ function ReplayContent({orgSlug, replaySlug, event}: Props) {
   }, [eventTimestamp, startTimestampMs]);
 
   if (fetchError) {
+    // TODO(replay): this is kind of a lot of writing. When the product docs launch in GA we will link to them instead.
+    const reasons = [
+      t('The Replay is still processing and is on its way'),
+      t('The Replay has been deleted'),
+      t('There is an internal systems error or active issue'),
+    ];
+
     return (
       <Alert type="info" showIcon data-test-id="replay-error">
-        {t('The replay associated with this event could not be found.')}
+        <p>
+          {t('The replay associated with this event could not be found.')}{' '}
```

Comment:
Link to https://docs.sentry.io/platforms/javascript/session-replay/#error-linking